OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs

```diff
@@ -303,21 +339,50 @@ private async void UpdateInitialVolume(object sender, MediaPlayerVolumeChangedEv
             });
         }
 
+        /// <summary>
+        /// Media playing event handler. Gets called when the media player state is changed to playing.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
         private async void MediaPlayer_Playing(object sender, EventArgs e)
         {
             Debug.WriteLine(DateTime.Now.ToString("hh:mm:ss.fff") + ": Media is playing");
+            if (_isFirstPlaying || _isReloading)
+            {
+                TrackDescription[] subtitleTracks = MediaTrackService.GetSubtitleTracks();
+                TrackDescription selectedSubtitleTrack = MediaTrackService.GetPreferredSubtitleTrack();
```

Comment:
These two methods call into VLC, while we are on the VLC event handler. See https://github.com/fossmium/OneDrive-Cloud-Player/issues/51

Reply:
What do you prefer how this is done instead?

Reply:
I will put them both in a thread pool

Reply:
Changed in commit c75747890a16727274a8939617e61444e5800603. Could you check if this change is following #51 LibVLCSharp's best practices?

Reply:
Now changed into a single task.run()